dev/io.openliberty.checkpoint_fat_mp/bnd.bnd

```diff
@@ -0,0 +1,145 @@
+#*******************************************************************************
+# Copyright (c) 2022, 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+	fat/src,\
```

Comment:
The source folder list here should match the `<classpathentry kind="src"/>` listed in `dev/io.openliberty.checkpoint_fat_mp/.classpath`